test/images/cuda-vector-add/BASEIMAGE

```diff
@@ -1,2 +1 @@
-amd64=nvidia/cuda:8.0-devel-ubuntu16.04
-ppc64le=nvidia/cuda-ppc64le:8.0-devel-ubuntu16.04
+amd64=nvidia/cuda:10.0-devel-ubuntu18.04
```

Comment:
Why have you removed the entry for ppc64le?

Reply:
The Cuda10 images for ppc64le doesn't seem to be available: https://hub.docker.com/r/nvidia/cuda-ppc64le/

Reply:
Why not retain ppcie with an older CUDA version? Also we need to set up continuous builds and update the test to use the latest image instead of having to manually update everytime a new cuda release is made.

Reply:
Automation can happen outside of this PR thought.

Reply:
Added back ppc64le with Cuda 9.2 and updated README accordingly.

Reply:
> The Cuda10 images for ppc64le doesn't seem to be available: https://hub.docker.com/r/nvidia/cuda-ppc64le/

Its not true, I see ppc image available and I can able to pull it as well.

```
root@localhost:~# docker pull nvidia/cuda-ppc64le:10.0-devel-ubuntu18.04
10.0-devel-ubuntu18.04: Pulling from nvidia/cuda-ppc64le
2c4fd606dee3: Pull complete
324762f962b9: Pull complete
31f2857f1765: Pull complete
8f3aa3f305b3: Pull complete
dc0ae3e48980: Pull complete
fc2724fdc475: Pull complete
a893585c19eb: Pull complete
13b584a89f22: Pull complete
Digest: sha256:0985f540c396e6ee0f158055ac573231040e2115d561f41cbc808fba60169c38
Status: Downloaded newer image for nvidia/cuda-ppc64le:10.0-devel-ubuntu18.04
root@localhost:~#
```

@jiayingz lets update ppc64le image as well.